hack/builder/version.sh

```diff
@@ -1,2 +1,2 @@
-VERSION=30-5.8.1
+VERSION=31
```

Comment:
I think we want `30- *.` From history I can see that the first number corresponds to fedora version...